public/pug/landing.pug

```diff
@@ -0,0 +1,16 @@
+extends layout
```

Comment:
Wie sorgt ihr dafür, dass eine Änderung an einem Template auch allen Clients bekannt gemacht wird? Jetzt ist es ja so, dass alles was im `public` liegt auch gecacht wird und damit die Änderungen nicht sofort sichtbar werden? Oder?

Reply:
Meinen Sie mit Änderungen neue Commits oder andere an das view übergebene Daten?

Reply:
Weder noch. Wenn man diese Datei ändert und die Anwendung neu deployt, mit dieser Änderung, kann das jeder Benutzer auch sofort sehen?
Beispiel: `a.blue-btn` wird zu `a.red-btn`.

Reply:
Wenn Sie das caching meinen: das liegt in diesem Fall in Herokus Händen.

Ansonsten weiß ich nicht worauf Sie hinaus wollen. Wenn ich das layout.pug template ändere dann wird die Änderung sofort angezeigt, da ja express aus den pug Dateien das html dynamisch generiert. Wir haben ja keine build pipeline die am Ende eine statische html Datei ausspuckt.

Reply:
Genau auf so eine `build pipeline` wollte ich hinaus. Die Information habe ich gebraucht bzw. wollte ich von euch hören, um es richtig bewerten zu können. Danke.